### PR TITLE
19.10 homepage and engage page

### PR DESCRIPTION
**Pull request by pmahnke**
## Done

- 19.10 homepage and engage page

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at:
    - http://0.0.0.0:8001/
    - http://0.0.0.0:8001/takeovers
    - http://0.0.0.0:8001/engage/19-10-webinar
    - http://0.0.0.0:8001/engage/
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- See that the page matches the [design](https://github.com/canonical-web-and-design/ubuntu.com/issues/5852) and the [brief](https://docs.google.com/spreadsheets/d/1SLmQWXO4P3iJdka4P0N1A95d755cLRIt68sxdocK7jw/edit#gid=0)

## Issue / Card

Fixes https://github.com/canonical-web-and-design/web-squad/issues/1820

### CODE REVIEW

File: templates/engage/19-10-webinar.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+   webinar_code: '<div class="jsBrightTALKEmbedWrapper" style="width:100%; height:100%; position:relative;background: #ffffff;"><script class="jsBrightTALKEmbedConfig" type="application/json">{ "channelId" : 6793, "language": "en-US", "commId" : 373550, "displayMode" : "standalone", "height" : "auto" }</script><script src="https://www.brighttalk.com/clients/js/player-embed/player-embed.js" class="jsBrightTALKEmbed"></script></div>'
+---
+
+Ubuntu 19.10 gives a boost to developer productivity through enhancements to Kubernetes and MicroK8s capabilities, ships with OpenStack Train updates that impact multi-cloud infrastructure economics and, as always, provides a streamlined desktop experience, including an experimental ZFS file system option.
+
+Highlights of the 19.10 release updates will be discussed between Ubuntu product, security and engineering teams in our upcoming webinar, ‘What’s new in Ubuntu 19.10’.
+
+In this webinar you will learn about:
+
+- New edge and AI capabilities for Kubernetes for a secure, production-grade Kubernetes environment
```
Comment (solazio) [inline templates/engage/19-10-webinar.md]: Here is a link missing for [Kubernetes](https://ubuntu.com/kubernetes)

File: templates/engage/19-10-webinar.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+- The inclusion of OpenStack Train updates, including live-migration to aid infrastructure operations
```
Comment (solazio) [inline templates/engage/19-10-webinar.md]: Missing link for [OpenStack](https://ubuntu.com/openstack)

**Comment by anasereijo** [comment]
@pmahnke please replace the image with the png version I sent you yesterday

**Comment by anasereijo** [comment]
@pmahnke  https://assets.ubuntu.com/v1/fae23a13-Eoan+Ermine-orange-outline.png

**Comment by anasereijo** [comment]
@pmahnke I'm having issues with the image (the overlay thing) so leave it as it is.

**Pull request merged**